client/src/components/navbar/index.js

```diff
@@ -36,10 +34,7 @@ const Navbar = props => {
           <FlatButton containerElement={<Link to="/" />} label="MESA" />
         </ToolbarGroup>
         <ToolbarGroup>
-          <div className="navbarSearch">
-            <TextField hintText="Search" />
-          </div>
-          <RaisedButton label="Search" />
+          <NavbarForm onSubmit={pushSearchResultsPage} />
```

Comment:
should this onSubmit prop have the same name as the prop name in the component itself (handleSubmit)?

Reply:
The handleSubmit prop is a part of the redux form API: https://redux-form.com/6.6.3/docs/api/props.md/